mailbox/store/src/main/java/org/apache/james/mailbox/store/GroupFolderResolver.java

```diff
@@ -19,23 +19,14 @@
 
 package org.apache.james.mailbox.store;
 
-import org.apache.james.mailbox.MailboxSession;
 import org.apache.james.mailbox.model.MailboxConstants;
 import org.apache.james.mailbox.store.mail.model.Mailbox;
 
 public class GroupFolderResolver {
-    
-    private final MailboxSession mailboxSession;
-
-    public GroupFolderResolver(MailboxSession mailboxSession) {
-        this.mailboxSession = mailboxSession;
-    }
 
     public boolean isGroupFolder(Mailbox mailbox) {
         String namespace = mailbox.getNamespace();
-        return namespace == null || 
-                (!namespace.equals(mailboxSession.getPersonalSpace())
-                && !namespace.equals(MailboxConstants.USER_NAMESPACE)
-                && !namespace.equals(mailboxSession.getOtherUsersSpace()));
```

Comment:
I don't understand how this case is covered by the new code, could you explain ?

Reply:
I got the impression to always repeat it.

If the mailbox is in the #private namespace  then it belongs to a user, then it is not a group mailbox.

The concept of OtherNamespace / personnalNamespace is a context depandant, IMAP specific designation of things. It is **orthogonal** to the MailboxPath object wich is AND should be**context indepandant**.

Reply:
As explained during the daily, as shared mailboxes are not supported, I'l remove the GroupFolderResolver and pass false where it was called, because we don't handle *shared mailboxes* for now.